.github/workflows/autofix.yml

```
@@ -0,0 +1,24 @@
+    branches:
+      - master
+permissions:
+  contents: read
+
+jobs:
+  autofix:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up environment
+        uses: ./.github/actions/setup
```

actionlint (1.7.7)

description is required in metadata of "Setup" action at "/home/jailuser/git/.github/actions/setup/action.yml"

(action)